src/inmanta/plugins.py

```diff
@@ -903,7 +922,7 @@ def wrapper(self, *args: object, **kwargs: object) -> Any:
             dictionary["__function__"] = fnc
 
             bases = (Plugin,)
-            fnc.__plugin__ = PluginMeta.__new__(PluginMeta, name, bases, dictionary)
+            fnc.__plugin__ = PluginMeta.__new__(PluginMeta, name, bases, dictionary)  # type: ignore[attr-defined]
```

Comment:
Should this rather be an entry in the mypy-baseline file?

Reply:
I think the ignore is appropriate in this case.